contracts/src/v0.8/tests/FunctionsBillingRegistryWithInit.sol

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.6;
+
+import "../dev/functions/FunctionsBillingRegistry.sol";
+
+contract FunctionsBillingRegistryWithInit is FunctionsBillingRegistry {
```

Comment:
Note: this is a wrapper for tests that aren't testing with the Upgrades plugin (e.g. Go tests)
They _could_ call `initialize` themselves, but this makes it cleaner.